atomic_reactor/plugins/post_tag_from_config.py

```diff
@@ -56,11 +56,11 @@ def get_tags(self):
         with open(tags_filename) as tags_file:
             for tag in tags_file:
                 tag = tag.strip()
-                if tag:
-                    if '-' in tag:
-                        self.log.warning("tag '%s' includes '-', ignoring", tag)
-                    else:
-                        tags.append(tag)
+                if is_valid_tag_name(tag) and '-' not in tag:
+                    tags.append(tag)
+                else:
+                    self.log.warning("tag '%s' does not match "
+                        "'[a-zA-Z0-9_][a-zA-Z0-9_.]{0,127}', ignoring", tag)
```

Comment:
I'd rather not have this regex re-defined here. Can we just make it available publicly from `util` module? Message would have to be written as "does not match regex, or includes dashes".